request.ts

```diff
@@ -4,6 +4,8 @@ import * as Promise from 'bluebird'
 import { PinejsClientCoreFactory } from 'pinejs-client-core'
 import TypedError = require('typed-error')
 
+export { PinejsClientCoreFactory } from 'pinejs-client-core'
```

Comment:
Can't this be `export const PinejsClientCoreFactory = typeof PinejsClientCoreFactory` given you import it above? (not sure if `typeof` is needed)

Reply:
It can be, but I think the `export .. from ..` is preferable